.github/workflows/sycl-linux-precommit.yml

```diff
@@ -72,6 +72,56 @@ jobs:
           else
             echo 'arc_tests="Matrix/"' >> "$GITHUB_OUTPUT"
           fi
+
+  split_test_build:
+    needs: [build]
+    if: ${{ always() && !cancelled() && needs.build.outputs.build_conclusion == 'success' }}
+    uses: ./.github/workflows/sycl-linux-run-tests.yml
+    with:
+      name: Build e2e tests
+      runner: '["Linux", "build"]'
+      image: ghcr.io/intel/llvm/ubuntu2204_intel_drivers:alldeps
+      image_options: -u 1001
+      target_devices: opencl:cpu
+      extra_lit_opts: --param test-mode=build-only
+      ref: ${{ github.sha }}
+      merge_ref: ''
+      sycl_toolchain_artifact: sycl_linux_default
+      sycl_toolchain_archive: ${{ needs.build.outputs.artifact_archive_name }}
+      sycl_toolchain_decompress_command: ${{ needs.build.outputs.artifact_decompress_command }}
```

Comment:
Don't remember why we needed this. Maybe because builds are release and we wanted some widely used format there. If that's the case, it's different for e2e tests and we can just hard-code the one that performs better.

Reply:
We also do this in the existing e2e pre-commit testing 
https://github.com/intel/llvm/blob/sycl/.github/workflows/sycl-linux-precommit.yml#L145-L147

Reply:
that reasoning sounds somewhat familiar

Reply:
Hard-coded the archive name as well as the compression/decompression commands.